src/issues/components/VotingResults.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <QList>
+    <QItem>
+      <QItemMain
+        :label="$t('ISSUE.VOTING.RESULTS.ENDED_AT', { date: $d(voting.expiresAt, 'long') })"
+        :sublabel="$t('ISSUE.VOTING.RESULTS.PARTICIPANTS', { number: voting.participantCount })"
+      />
+    </QItem>
+    <template
+      v-if="!isCancelled"
+    >
+      <QItem
+        dense
+        class="justify-end"
+      >
+        <QItemSide
+          right
+          :stamp="$t('ISSUE.VOTING.RESULTS.TOTAL_SCORE')"
+        />
+      </QItem>
+      <QItem
+        v-for="{ id, icon, label, sumScore} in options"
+        :key="id"
+      >
+        <QItemSide
+          :icon="icon"
+          color="primary"
+        />
+        <QItemMain>
+          {{ label }}
+        </QItemMain>
+        <QItemSide stamp>
+          {{ sumScore }}
+        </QItemSide>
+      </QItem>
+    </template>
+    <QItem
+      v-else
```

Comment:
A tie seems to also count as isCancelled and thus triggers the undecided messages:
![image](https://user-images.githubusercontent.com/17573771/54434537-cf177700-472e-11e9-8fe5-df653932e5da.png)

I think it would be cooler to have a separate message for ties as well. Maybe we compare the sumScores and make another computed property to be able to have one more special case in the template?

Reply:
Found it! Historic votings received the status of the issue, but they are not affected by it. `isCancelled` should always be `false` for historic votings.